sda-commons-server-auth-testing/README.md

```diff
@@ -83,7 +83,76 @@ Examples can be found in the [test source branch](./src/test) of this module. Th
 - [An example app](./src/test/java/org/sdase/commons/server/auth/testing/test/AuthTestApp.java)  
 - [A test with authentication](./src/test/java/org/sdase/commons/server/auth/testing/AuthRuleIT.java)  
 - [A test with disabled authentication](./src/test/java/org/sdase/commons/server/auth/testing/AuthDisabledIT.java)
-  which uses the `EnvironmentRule` instead of the `AuthRule`
+- [An appropriate test `config.yaml`](./src/test/resources/test-config.yaml)
+
+## Auth Extension
+The `AuthExtension` puts the `AuthConfig` in an environment variable named `AUTH_RULE` (for backwards compatibility).
+The configuration in the test needs to use this property and the application is required to use the
+[`ConfigurationSubstitutionBundle`](../sda-commons-server-dropwizard/src/main/java/org/sdase/commons/server/dropwizard/bundles/ConfigurationSubstitutionBundle.java)
+from [`sda-commons-server-dropwizard`](../sda-commons-server-dropwizard/README.md):
+
+```java
+public class MyApp extends Application<MyConfig> {
+
+   @Override
+   public void initialize(Bootstrap<MyConfig> bootstrap) {
+      bootstrap.addBundle(ConfigurationSubstitutionBundle.builder().build());
+      bootstrap.addBundle(AuthBundle.builder().withAuthConfigProvider(MyConfig::getAuth).build());
+   }
+
+   @Override
+   public void run(MyConfig configuration, Environment environment) {
+      // ...
+   }
+}
+```
+
+```yaml
+# test-config.yaml
+server:
+  applicationConnectors:
+  - type: http
+    port: 0
+  adminConnectors:
+  - type: http
+    port: 0
+
+# The configuration of the test auth bundle is injected here
+auth: ${AUTH_RULE}
+```
+
+To implement the test, the `AuthExtension` has to be initialized before the `DropwizardAppExtension`:
+
+```java
+class AuthExtensionIT {
+
+  @RegisterExtension public static final AuthExtension AUTH = AuthExtension.builder().build();
```

Comment:
Not 100% sure if we also need `@Order`. I think we get a reproducible but random order otherwise.

Reply:
we clarified, that we need @order

Reply:
Could you please add `@Order` to the two extensions of the example?